tests/basic/misc_test.go

```diff
@@ -713,3 +715,86 @@ var _ = Describe("{SecretsVaultFunctional}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{VolumeCreatePXRestart}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("VolumeCreatePXRestart", "Validate restart PX while create and attach", nil, 0)
+
+	})
+	contexts := make([]*scheduler.Context, 0)
+
+	stepLog := "Validate volume attachment when px is restarting"
+	It(stepLog, func() {
+		var createdVolIDs map[string]string
+		var err error
+		volCreateCount := 10
+		stepLog := "Create multiple volumes , attached and restart PX"
+		Step(stepLog, func() {
+			dash.Infof(stepLog)
+
+			stNodes := node.GetStorageNodes()
+			index := rand.Intn(len(stNodes))
+			selectedNode := stNodes[index]
+
+			dash.Infof("Creating and attaching %d volumes on node %s", volCreateCount, selectedNode.Name)
+
+			wg := new(sync.WaitGroup)
+			wg.Add(1)
+			go func(appNode node.Node) {
+				createdVolIDs, err = CreateMultiVolumesAndAttach(wg, volCreateCount, selectedNode.Id)
+				if err != nil {
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Error while creating volumes. Err: %v", err))
+				}
+			}(selectedNode)
+			time.Sleep(2 * time.Second)
```

Comment:
can we avoid static sleep?

Reply:
this is just to make sure pxrestart in. next step happens after 1 or volumes are created